### ISSUE TITLE
Unsupported version with codnd 4.0

### ISSUE THREAD

**Issue by ishit93** [conda/conda#2186]
I got this kind of dependency error after updating python library. 
Can any body tell me How to go previous dependencies?

```
Fetching package metadata: ....
Solving package specifications: .An unexpected error has occurred, please consider sending
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Fetching package metadata: ....
Solving package specifications: ....................
Package plan for installation in environment C:\Anaconda:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pyyaml-3.11                |           py27_3         168 KB
    conda-4.0.0                |           py27_0         227 KB
    ------------------------------------------------------------
                                           Total:         395 KB

The following packages will be UPDATED:

    conda:  3.19.3-py27_0 --> 4.0.0-py27_0
    pyyaml: 3.11-py27_2   --> 3.11-py27_3

Proceed ([y]/n)?


Fetching packages ...
pyyaml-3.11-py 100% |###############################| Time: 0:00:00 734.21 kB/s
conda-4.0.0-py 100% |###############################| Time: 0:00:00 711.79 kB/s
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
[      COMPLETE      ]|##################################################| 100%
```

Error which I got.
```
Fetching package metadata: ....
Solving package specifications: .An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Anaconda\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Anaconda\lib\site-packages\conda\cli\main.py", line 194, in main
    args_func(args, p)
  File "C:\Anaconda\lib\site-packages\conda\cli\main.py", line 201, in args_func
    args.func(args, p)
  File "C:\Anaconda\lib\site-packages\conda\cli\main_create.py", line 49, in execute
    install.install(args, parser, 'create')
  File "C:\Anaconda\lib\site-packages\conda\cli\install.py", line 334, in install
    minimal_hint=args.alt_hint)
  File "C:\Anaconda\lib\site-packages\conda\plan.py", line 403, in install_actions
    config.track_features, minimal_hint=minimal_hint):
  File "C:\Anaconda\lib\site-packages\conda\resolve.py", line 747, in solve
    minimal_hint=minimal_hint)
  File "C:\Anaconda\lib\site-packages\conda\resolve.py", line 492, in solve2
    clauses = set(self.gen_clauses(v, dists, specs, features))
  File "C:\Anaconda\lib\site-packages\conda\resolve.py", line 350, in gen_clauses
    assert len(clause) > 1, '%s %r' % (fn1, ms)
AssertionError: qt-4.8.7-vc10_5.tar.bz2 MatchSpec(u'openssl 1.0.2*')
```

**Comment by kalefranz** [comment]
@mcg1969 Have we fixed this assertion error? I think there are cases where old inconsistent environments can have multiple versions of a package installed. I'm guessing that's what's being run into here?

**Comment by mcg1969** [comment]
No, I have not attempted to diagnose this.

**Comment by kalefranz** [comment]
@ishit93 I think that error has been fixed as of `4.0.2`.  Can you update conda (`conda update conda`) and re-open this issue if not?
